rtcstats.js

```diff
@@ -30,6 +30,7 @@
   function trace() {
     //console.log.apply(console, arguments);
     // TODO: drop getStats when not connected?
+    arguments.push(new Date().getTime());
```

Comment:
argh. the server side assumes this to be an object right now, in particular when setting the server side timesttamp :-(